Docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:trusty
+
+RUN apt-get update
+RUN apt-get -y install software-properties-common python-software-properties
+RUN add-apt-repository ppa:git-core/ppa
+RUN apt-get -y install git
+RUN apt-get -y install sudo
+RUN apt-get -y install wget
+
+RUN git clone -b docker https://github.com/JGulbronson/rmc.git ~/rmc
```

Comment:
That's just used to create the image.. so on Travis we should still be able to clone their branch and load it in to the Docker instance. This will just use master for creating the image itself (I think)